wiremock/stubs/application-statuses.json

```diff
@@ -3,49 +3,131 @@
     "statusDetails": []
   },
+  {
+    "id": "9a381bc6-22d3-41d6-804d-4e49f428c1de",
+    "name": "offerDeclined",
+    "label": "Offer declined or withdrawn",
+    "description": "The accommodation offered has been declined or withdrawn.",
+    "statusDetails": [
+      { "id": "62645779-242d-4601-a8f8-d2cbf1d41dfa", "name": "areaUnsuitable", "label": "Area unsuitable" },
+      {
+        "id": "fc38f750-e9d2-4270-b542-d38286b9855c",
+        "name": "changeOfCircumstances",
+        "label": "Change of circumstances"
+      },
+      { "id": "31122b89-e087-4b5f-b59a-f7ffa0dd3e0c", "name": "noResponse", "label": "No response" },
+      {
+        "id": "32e62af3-6ea5-4496-a82c-b7bad67080a5",
+        "name": "offerWithdrawnByNacro",
+        "label": "Offer withdrawn by Nacro"
+      },
+      {
+        "id": "b5bfbff4-aaa6-4fb0-ba36-5bca58927dc5",
+        "name": "propertyUnsuitable",
+        "label": "Property unsuitable for needs"
+      },
+      { "id": "f58267b8-f91b-4a4f-9aa2-80089ba111e4", "name": "withdrawnByReferrer", "label": "Withdrawn by referrer" },
+      {
+        "id": "ed5d529c-d7d1-4f29-a0c0-89fd104cc320",
+        "name": "rehousedByAnotherLandlord",
+        "label": "Rehoused by another landlord"
+      }
+    ]
+  },
   {
     "id": "004e2419-9614-4c1e-a207-a8418009f23d",
     "name": "withdrawn",
-    "label": "Withdrawn",
-    "description": "The application was withdrawn by the referrer, or MoJ staff.",
+    "label": "Referral withdrawn",
+    "description": "The prison offender manager (POM) withdrew the application.",
+    "statusDetails": [
+      {
+        "id": "e4a2391e-e847-427a-a913-51e0b0ad9f52",
+        "name": "hdcNoLongerEligible",
+        "label": "HDC - No longer eligible"
+      },
+      {
+        "id": "c5dce0d2-fc05-4a07-8157-25b8821cdb06",
+        "name": "governorDecidedUnsuitable",
+        "label": "Governor - Decided unsuitable"
+      },
+      {
+        "id": "3fb37f85-be88-4eee-812d-af122e268eef",
+        "name": "governorChosenAlternative",
+        "label": "Governor - Chosen alternative accommodation"
+      },
+      { "id": "8a857a7d-94f9-43ec-963c-2a2528e88a6e", "name": "governorOther", "label": "Governor - Other" },
+      {
+        "id": "d3c789b8-947d-4e24-9cef-335545d85abe",
+        "name": "withdrewOrDeclinedOffer",
+        "label": "Withdrew or declined offer"
+      }
+    ]
+  },
+  {
+    "id": "f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9",
+    "name": "cancelled",
+    "label": "Referral cancelled",
+    "description": "The application has been cancelled.",
+    "statusDetails": [
+      { "id": "f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9", "name": "assessedAsHighRisk", "label": "Assessed as High Risk" },
+      { "id": "f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9", "name": "notEligible", "label": "Not Eligible" },
+      {
+        "id": "f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9",
+        "name": "noRecourseToPublicFunds",
+        "label": "No Recourse to Public Funds"
```

Comment:
just ran this by Chanel - can we only capitalise the first word for these?

Reply:
Ah, we can but the statuses are coming from the API. I can make a follow-up card for this to be changed in the API and I'll ask Chanel if this is a blocker to launching.

Reply:
I have a PR open here: https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1489